IKVM.0.47.0.1.nuspec

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>IKVM</id>
+        <version>0.47.0.1</version>
+        <authors>Ullink</authors>
+        <owners>Ullink</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>IKVM 0.47 compiled with Ullink Key</description>
+        <summary />
+        <language />
+    </metadata>
+    <files>
+        <file src="bin\IKVM.AWT.WinForms.dll" target="lib\net35\IKVM.AWT.WinForms.dll" />
```

Comment:
Should bi net20, no?

Reply:
It seems we already bundled it like that. For the sake of starting from exactly where we are, we can keep it like this :)